commit a735c66ecbce5f2cc23a89095945fc5eda02eef1 (refs/changes/40/2940/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2017-11-14T21:31:38+00:00 (1 year, 11 months ago)
    
    VOLAPI-91 want experimental NFS volumes tests in nightly-1
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 7205cea..f525f5f 100644
--- a/Makefile
+++ b/Makefile
@@ -73,6 +73,7 @@ release: all deps docs $(SMF_MANIFESTS)
     $(ROOT)/package.json \
     $(ROOT)/sapi_manifests \
     $(ROOT)/smf \
+    $(ROOT)/test \
     $(ROOT)/tools \
     $(RELSTAGEDIR)/root/opt/smartdc/$(SERVICE_NAME)/
 	mkdir -p $(RELSTAGEDIR)/root/opt/smartdc/boot
